test/python/pulse/_dummy_targets.py

```diff
@@ -0,0 +1,36 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2024
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Pulse target for unit tests."""
+
+from qiskit.pulse.compiler.target import QiskitPulseTarget
+
+
+TWOQ_CROSSRES_TARGET = QiskitPulseTarget(
```

Comment:
It is odd that this looks like it is here to be used for tests but then the tests define a different example target?

Reply:
Test target includes some fancy settings to cover edge cases. This will be used for the test of future passes. Currently we don't have any pass that requires hardware information.